scripts/dtc_installer.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+DIR="$PWD"
+
+check_dpkg () {
+	LC_ALL=C dpkg --list | awk '{print $2}' | grep "^${pkg}" >/dev/null || deb_pkgs="${deb_pkgs}${pkg} "
+}
+
+dtc_git_build () {
+	project="dtc"
+	server="git://git.kernel.org/pub/scm/utils/dtc/dtc.git"
+
+	if [ ! -d ${HOME}/git/ ] ; then
+		mkdir -p ${HOME}/git/ || true
+	fi
+
+	if [ ! -f ${HOME}/git/${project}/.git/config ] ; then
+		git clone ${server} ${HOME}/git/${project}/
+	fi
+
+	cd ${HOME}/git/${project}/
+	make clean
+	git checkout master -f
+	git pull || true
+
+	if [ ! "x${git_tag}" = "x" ] ; then
+		test_for_branch=$(git branch --list ${git_tag}-build)
+		if [ "x${test_for_branch}" != "x" ] ; then
+			git branch ${git_tag}-build -D
+		fi
+
+		git checkout ${git_tag} -b ${git_tag}-build
+	fi
+
+	make clean
+	make PREFIX=/usr/local/ CC=gcc CROSS_COMPILE= all
```

Comment:
```suggestion
	make PREFIX=/usr/local/ CC=gcc CROSS_COMPILE= all NO_PYTHON=1
```

This makes it so it won't attempt to build the Python library that fails